dir.props

```diff
@@ -117,6 +117,7 @@
     <DnuRestoreDir Include="$(ToolsDir)" />
     <!-- workaround to address issue where DNX won't recurse directories under a project.json
          https://github.com/aspnet/dnx/commit/0dda8bf86863364cc20421f1af7494f1b2a3256f -->
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\*\src\**\project.json" />
```

Comment:
With this I don't think we need the netcore50aot folder entry that @dagood provided below.